api/ereqs_admin/tests/admin_tests.py

```diff
@@ -85,6 +85,7 @@ def test_user_edit_max(admin_client):
     assert user.first_name == 'posttest'
 
 
+@pytest.mark.skip(reason="We've temporarily removed pdf upload")
```

Comment:
Wait, we _supported_ pdf upload?! 😂

Reply:
Heh, indeed, although just in the file sense, without any import-and-convert aspect.